portfolio.html

```diff
@@ -8,11 +8,96 @@
     <link rel="stylesheet" type="text/css" href="normalize.css">
     <link rel="stylesheet" type="text/css" href="portfolio.css">
     <link href="https://fonts.googleapis.com/css?family=Dosis" rel="stylesheet">
-    <title>Document</title>
+    <title>Portfolio</title>
     
 </head>
+
 <body>
 
+
+<!--------Navbar-------->
+
+    <!--------Video-------->
+
+    <div class="video_container">
+        <video src="video/99544-1080.mp4" type="video/mp4" loop autoplay></video>
+    </div>
+
+    <nav class="nav_container">
+        <div class="personal_logo">
+            <img class="personal_logo" src="images/logo_transparent.png" alt="personal logo">
+        </div>
+
+        <div class="nav_bar">
+            <a href="#about_me" title="Link to about me section">About Me</a>
```

Comment:
do you need to use ids here? if they're for styling, consider changing to classes

Reply:
I used them to point to section in the page